src/axom/mint/utils/ExternalArray.hpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef MINT_EXTERNALARRAY_HPP_
+#define MINT_EXTERNALARRAY_HPP_
+
+#include "axom/core/Array.hpp"  // to inherit
+#include "axom/core/Types.hpp"
+
+#include "axom/fmt.hpp"
+
+#include "axom/slic/interface/slic.hpp"  // for slic logging macros
+
+namespace axom
+{
+namespace mint
+{
+/*!
+ * \class ExternalArray
```

Comment:
Design questions:
* Should this class live in `mint`, or is it sufficiently useful to warrant moving to `core`. My main reservation is about the fact that it does not delete its memory, which might make it easy to misuse
* Should this class extend `axom::Array` or `axom::ArrayBase`? Obviously, extending the former allows it to be so concise.

Reply:
Please add a note about how this differs from `axom::ArrayView`